### ISSUE TITLE
Doc page cleanup

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#206]
Would be good to have all the doc controls in one area:

* Move 'City Council Meeting Agendas' button to left sidebar where download button is. 
* Remove 'Back to Documents' button or make breadcrumb.
* Remove the file name (ex: 03_08_2016_revised.pdf)
* Download button could be smaller

Reference: https://www.westcarrollton.org/documents/march-8-2016-city-council-meeting-2/?filter_categories[]=79

**Comment by lukefretwell** [comment]
Seems like there's a better UI for the 'Document Library.' 

Perhaps there should be space above the agencies for icons or default agencies, city council to icons. 

Just looks odd where it is and that it's not either a search filter or part of navigation.

**Comment by jlyon** [comment]
> Seems like there's a better UI for the 'Document Library.' 

I think we should probably match the ui for agency subpages: https://www.westcarrollton.org/city-council-agendas/. I'll play with this in the pattern library during the next sprint.

**Comment by jlyon** [comment]
@lukefretwell what do you think of this?
before: https://www.westcarrollton.org/documents/lawn-care-bid-packet/
after: http://pattern.getproudcity.com/?v=&page=document-content

**Comment by lukefretwell** [comment]
@jlyon looks cleaner, but the multiple fonts are a little distracting. 

Recs:

* use <i class="fa fa-download"></i> for download icon (that way it will be universal on any download activity, regardless of file type - just noticed you do here: http://pattern.getproudcity.com/?page=document-list
* The tags are really great, but think we should figure out a way to standardize where/how they're displayed. Like a 'Related' subhead below the information specific to the page (also for individual people pages). 
* Would spell out month in date (like you've done here: https://www.westcarrollton.org/city-council-agendas/)

**Comment by jlyon** [comment]
Changed date to be full month and pushed this for now.

**Other items:**

Icons: created another issue to tackle this #236.  Left the download icon as is on this pg. Maybe we can add the filetype icon to the full doc view after #236 is done (maybe in the title)

> The tags are really great, but think we should figure out a way to standardize where/how they're displayed

This is a bigger item that will require discussion, so I created #237

@lukefretwell
